### ISSUE TITLE
Add BOT for pirate island

### ISSUE THREAD

**Issue by shobhitagarwal1612** [Software-Incubator/DirectMe#23]
BOT should undock unfortunate ported ships on pirate island

**Comment by utkbansal** [comment]
Depends on #79

**Comment by apoorvaeternity** [comment]
It is a luck based thing.Maybe instead of complicating things we can just decide beforehand whether the person who is docking the ship on the pirate island will get caught by the pirates or not.

**Comment by utkbansal** [comment]
I don't think it would be a good idea to change rules at this point without proper discussions.
